rest/rest-server/src/test/java/functionaltests/utils/RestFuncTUtils.java

```diff
@@ -49,11 +49,10 @@ private RestFuncTUtils() {
     }
 
     public static void destroy(Process process) throws Exception {
-        process.destroy();
         close(process.getOutputStream());
         close(process.getInputStream());
         close(process.getErrorStream());
-        process.waitFor();
+        process.destroyForcibly();
```

Comment:
Why did you have to change the destroy and wait behavior? 
Could you explain what the change implies here?

Reply:
In Windows process.waitFor() causes an issue and blocks the destruction process. So I used destroyForcibly(), as it is efficient and occurs after closing all the PA process streams.